yt/yt/server/controller_agent/controllers/operation_controller_detail.cpp

```diff
@@ -9682,6 +9682,10 @@ void TOperationControllerBase::InitUserJobSpecTemplate(
         jobSpec->add_environment(Format("YT_DISCOVERY_ADDRESSES=%v", addresses));
     }
 
+    if (jobSpecConfig->DockerImage) {
```

Comment:
nit: `if (const auto& dockerImage = jobSpecConfig->DockerImage)` may be a little bit shorter. Not important, of course.